secret_store/src/helpers.rs

```diff
@@ -0,0 +1,12 @@
+use ethcore::client::{Client, BlockChainClient, BlockId};
```

Comment:
Could you please add license preamble to the beginning of the file?

Reply:
Or maybe just make this function a member of `TrustedClient`? Both have similar purpose.

Reply:
Don't think so - guess it will be enough for new file only. Thanks!

Reply:
Okay I'll send a separate PR to change license preamble.

Reply:
Okay. Got it!